### ISSUE TITLE
Separating repositories

### ISSUE THREAD

**Issue by martinbeentjes** [PocketServerMP/PocketServer#57]
I have been looking into the repository today and the first thing I saw, was that multiple projects were bundled into one big repository.

Why did you guys make the decision to not keep things like the Server and API apart from each other in separate repositories and thus keeping it a bit future proof (if it'll be flooded with PR's/issues)

**Comment by Serializator** [comment]
:+1:

**Comment by j0ach1mmall3** [comment]
:+1:

**Comment by TheLightMC** [comment]
I'm fine with separating the repository however I find it much easier than managing two different repositories especially while we do major changes during our development of the API. However I'm fine with doing whatever people would prefer. @Fireflies any opinion on splitting the repository?

**Comment by nickrobson** [comment]
Personally I'm impartial to this, because if the system ain't broke, why fix it?

**Comment by Serializator** [comment]
@nickrobson There's a difference between fixing something and improving something.

**Comment by martinbeentjes** [comment]
It is not broken, but I find it more reasonable to have two separate repositories for the server and api in stead of putting them both in one repository. Therefore you can keep track both development timelines apart from each other.

All issues regarding the API or Server should be tagged with 'server' or 'api' at this moment if you want to keep visible to which project it is subjected to.

**Comment by Serializator** [comment]
:+1:

**Comment by Fireflies** [comment]
Far larger projects use a similar system and their maintainers manage to keep their heads on whilst doing it. If we're using a fancy tool to manage things (maven) then we may as well take full advantage of it, especially when it makes development far easier.

We may switch in the future, although it is far more likely we'll simple enforce issue labelling and discard or ignore unlabelled issues.

tldr; if it can work for bigger projects it works for us.
